presto-docs/src/main/sphinx/functions/session-info.rst

```diff
@@ -0,0 +1,14 @@
+============================
```

Comment:
The bar width should match the text

Reply:
Let's name the file `system` so it's shorter (the additional `-info` part doesn't seem to add any value)